src/index.css

```diff
@@ -5,9 +5,25 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+background-color: lightblue;
+
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
+    
+}
+
+h1{
+  font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  color: green;
+  text-transform: uppercase;
+  text-decoration: underline;
+  margin-bottom: 80px;
+  margin-top: 50px;
+  font-size: 52px;
 }
+
+
```

Comment:
maybe add some media queries for the larger screens, the smaller screen actually doesn't look to bad below 640px. also add some padding for in between memes to make it clearer to see them!